lib/js/components/Buttons/IconButton/IconButton.vue

```diff
@@ -95,48 +99,26 @@
 
 			&:hover,
 			&.-hovered {
-				#{$self}__button {
-					background-color: map-get($color-map, 'filled', 'background-hovered');
-
-					&.-outlined {
-						background-color: map-get($color-map, 'outlined', 'background-hovered');
-						color: map-get($color-map, 'inverted', 'icon');
-					}
-				}
 				#{$self}__label {
 					color: map-get($color-map, 'filled', 'color-hovered');
 				}
 			}
 
 			&:focus,
 			&.-focused {
-				#{$self}__button {
-					background-color: map-get($color-map, 'filled', 'background-focused');
-
-					&.-outlined {
-						background-color: map-get($color-map, 'outlined', 'background-focused');
-						color: map-get($color-map, 'inverted', 'icon');
-					}
-				}
 				#{$self}__label {
 					color: map-get($color-map, 'filled', 'color-focused');
 				}
 			}
 
 			&.-disabled {
 				#{$self}__button {
-					background-color: map-get($color-map, 'filled', 'background-disabled');
-
 					&.-outlined {
-						background-color: map-get($color-map, 'outlined', 'background');
-						border-color: map-get($color-map, 'outlined', 'disabled', 'border');
-
 						.a-iconButton__icon {
 							color: map-get($color-map, 'outlined', 'disabled', 'icon');
```

Comment:
Finally we are not touching `__button` but need touch this class, because icon in IconButton in Button is different element than icon in Button component.